platforms/core-execution/execution/src/main/java/org/gradle/internal/execution/steps/AbstractResolveCachingStateStep.java

```diff
@@ -67,12 +67,17 @@ public CachingResult execute(UnitOfWork work, C context) {
         return new CachingResult(result, cachingState);
     }
```

Comment:
:x: Let's move the logging to where it belongs.

Reply:
As discussed, the logging seems to be fine as is.